### ISSUE TITLE
[BUG] Chromium binary is not signed

### ISSUE THREAD

**Issue by tymfear** [microsoft/playwright#10642]
**Context:**
- Playwright Version: [what Playwright version do you use?]: 1.17
- Operating System: [e.g. Windows, Linux or Mac]: MAc
- Node.js version: [e.g. 12.22, 14.6]: 14.17.0
- Browser: [e.g. All, Chromium, Firefox, WebKit]: Chromium
- Extra: [any specific details about your environment]

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```bash
$: npx playwright install chromium
$: codesign -dvv Chromium.app 
Chromium.app: code object is not signed at all
```

**Describe the bug**
Our security application blocks unsigned applications from reading directories and other things.
So basically it is quite hard to use PW as it asks for permissions for everything (literally hundreds of requests).

**Comment by tymfear** [comment]
Btw, I haven't checked other binaries as we currently don't use them, but if this is considered a fixable issue, then probably others should be signed as well.

**Comment by pavelfeldman** [comment]
If you need a signed binary, you can use one of the official browsers: Google Chrome or Microsoft Edge. Set the channel property in your test config or when you are launching the browser to 'chrome' or 'msedge' and it will use the local branded version.

**Comment by tymfear** [comment]
Hmm, we rely on PW provided browsers. But ok, we got a workaround.
Just out of curiosity is it somehow complicated thing to sign binaries? Never did that so have no idea what it takes to do it.

**Comment by pavelfeldman** [comment]
I don't know if it is, but we've never been asked to before, so why bothering...

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.
